bloom-housing/bloom
# feat: 3909/add redirect url prisma

# Pull Request Template

## Issue Overview
This PR implements the Mandated Accounts changes so it is compatible with Prisma

This PR addresses #3909


## Description
- Added previous backend changes for Mandated Accounts to the Prisma email service and user service
- Handle undefined urls
- Add tests 

## How Can This Be Tested/Reviewed?

Ensure you are running on the Prisma version of Bloom Core
Perform Create Account and Forgot Password flow and ensure everything works properly
With MandatedAccounts feature flag enabled: When logged out, go to a listing and hit apply online
- Forget Password flow
- Create Account flow

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
